raven/base.py

```diff
@@ -636,7 +637,10 @@ def capture(self, event_type, data=None, date=None, time_spent=None,
             **kwargs)
 
         # should this event be sampled?
-        if self._random.random() < self.sample_rate:
+        if not sample_rate:
```

Comment:
right, so if I just change this to `if sample_rate is None`, that achieves these cases:

sample_rate=None -> default to client.sample_rate
sample_rate=any numeric value -> overwrite client.sample_rate
sample_Rate=1.0 -> disable sampling


Does that sound OK?